craigslist-apartments.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+let axios = require('axios');
+let cheerio = require('cheerio');
+let filters = require('./filters');
+
+const Filter = new filters();
+let mountedURL = mountURL(Filter); 
+
+function mountURL(filter) {
```

Comment:
If this is the place you are assembling the url, you should receive an extra parameter `startIndex`. You'll use it to add `&s=${startIndex}` to the url.